apps/web/pages/d/[link]/book.tsx

```diff
@@ -74,7 +74,7 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
     ...eventTypeRaw,
     metadata: EventTypeMetaDataSchema.parse(eventTypeRaw.metadata || {}),
     recurringEvent: parseRecurringEvent(eventTypeRaw.recurringEvent),
-    bookingFields: eventTypeBookingFields.parse(eventTypeRaw.bookingFields || []),
+    bookingFields: getBookingFieldsWithSystemFields(eventTypeRaw),
```

Comment:
Making this change here, now ensures that system fields are always sent from backend.